src/MatrixClientPeg.js

```diff
@@ -111,6 +126,14 @@ class MatrixClientPeg {
         if (localStorage) {
             opts.sessionStore = new Matrix.WebStorageSessionStore(localStorage);
         }
+        if (window.indexedDB && localStorage) {
+            opts.store = new Matrix.IndexedDBStore(
+                new Matrix.IndexedDBStoreBackend(window.indexedDB),
```

Comment:
While we're at it, I just spent 5 minutes being confused by the fact that the js-sdk defaults the store to 2 different things in 2 different places (StubStore in client.js but MatrixInMemoryStore in src/matrix.js).